performance/cleanup_resource/cleanup_resource.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"database/sql"
+	"log"
+	"os"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/cloudspannerecosystem/harbourbridge/performance"
+)
+
```

Comment:
Please add a doc/comment on the purpose of this module

Reply:
added comments